Author: Joachim Reichel <reichel@debian.org>
Description: Detect encoder and decoder dynamically at runtime. This results in
better defaults depending on the user's environment and avoids data loss if no
suitable encoder/decoder is found.
--- a/src/normalize-mp3.in
+++ b/src/normalize-mp3.in
@@ -26,12 +26,12 @@
 # %b becomes bitrate of re-encoded file, as specified by the -b option
 # Example: $OGGENCODE="oggenc -Q -b %b -o %m %w"
 
-$MP3DECODE  = "@MP3DECODE@";
-$MP3ENCODE  = "@MP3ENCODE@";
-$OGGDECODE  = "@OGGDECODE@";
-$OGGENCODE  = "@OGGENCODE@";
-$FLACDECODE = "@FLACDECODE@";
-$FLACENCODE = "@FLACENCODE@";
+# $MP3DECODE  = "@MP3DECODE@";
+# $MP3ENCODE  = "@MP3ENCODE@";
+# $OGGDECODE  = "@OGGDECODE@";
+# $OGGENCODE  = "@OGGENCODE@";
+# $FLACDECODE = "@FLACDECODE@";
+# $FLACENCODE = "@FLACENCODE@";
 
 # The %w etc. substitutions should *not* be used in the following, as
 # this script knows about their options already.
@@ -614,7 +614,7 @@
 	    }
 
 	    unless ($encoder) {
-		print STDERR "$progname: $output_file: no decoder available\n";
+		print STDERR "$progname: $output_file: no encoder available\n";
 		print STDERR "$progname: leaving output in $tmp_file\n";
 		next;
 	    }
